languages/csharp/exercises/concept/structs/Structs.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+
+public struct Coord
+{
+    public Coord(ushort x, ushort y)
```

Comment:
Making the coordinate a `ushort` does mean taking a dependency on the `integral-numbers` exercise. You could consider making it an `int`, or do you feel the `ushort` is better here?

Reply:
I favor `ushort` as it puts the struct within the 16 byte limit which gets talked about quite a bit.

Reply:
I've added `integral-numbers` as a prerequisite.